pkg/controller/sinks/sinks_test.go

```diff
@@ -151,23 +151,7 @@ func getAddressable_noStatus() *unstructured.Unstructured {
 	}
 }
 
-func getAddressable_nilAddress() *unstructured.Unstructured {
```

Comment:
Was this not used?

Reply:
If so, does that mean we are missing a test?

Reply:
The function was not called. Here's what that test would look like: https://github.com/knative/eventing-sources/pull/316. I can wait for that PR to be merged in and then fix the name change in this PR.